README.md

```diff
@@ -224,10 +224,12 @@ bandwidth. To avoid this, you can use the "partial clone" feature by replacing
 the steps with this:
 
 ```yaml
+      - name: Configure Git
+      - run: git config --global http.https://github.com/.extraheader "Authorization: Basic $(echo -n x-access-token:${{ github.token }} | base64 --wrap=0)"
```

Comment:
```suggestion
      - run: git config --global http.https://github.com/.extraheader "Authorization: Basic $(echo -n 'x-access-token:${{ github.token }}' | base64 --wrap=0)"
```

I thought about this last time but forgot to mention it, seems like it's a good idea to quote it just in case. My personal style is to quote things aggressively because who knows, but in this case I'm also not sure if we can count on the GH token not containing shell characters.